pelicun/settings/schema.json

```diff
@@ -0,0 +1,451 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "Options": {
```

Comment:
I don't recognize this Options/Sampling/SampleSize part. Where and how is this used instead of the sample size specification within the `DL` part?

Reply:
Since it does not say `required`, I assume it is not necessary to have this `Options` part. So, I assume you are checking that the sample size is an integer here. I think we discussed relaxing this need and allowing both integer and string and you probably have that in a future commit.

I still don't understand why we would have this info under Options/Sampling/SampleSize

Reply:
This path was incorrect and is removed with 25ac3563. The most recent version of the branch has all `number` and `integer` entries under `anyOf` definitions, allowing `string` as well.

Reply:
Perfect, thanks.